Frontend/ui-customer/src/Screens/ZooTicketBookingFlow/TicketBookingScreen/TicketInfoScreen/View.purs

```diff
@@ -280,12 +293,12 @@ ticketImageView state push =
           , height $ V 150
           , gravity CENTER
           , onClick push $ const IncrementSliderIndex
+          , clickable $ if state.props.rightButtonDisable then false else true
```

Comment:
use `not state.props.rightButtonDisable`

Reply:
done